articles/automated-fake-news-detection/index.md

```diff
@@ -0,0 +1,130 @@
+
+### Automated Fake News Detection
+
+Automated detection systems provide value in terms of automation and scalability. There are various techniques and approaches implemented in fake news detection research. And it is worth noting that these approaches often overlap depending on perspective. From my personal perspective, I choose to discuss only two approaches. These two approaches focus on the methods used, as opposed to the content being analysed. They may also both involve Natural Language Processing (NLP) in their methodology.
+Natural Language Processing enables computers to understand natural/human language and respond appropriately. Hence, there are two aspects involved:
+· Natural Language Understanding
+· Natural Language Generation
+The two approaches to fake news detection are:
+· Machine Learning approach
+· Deep Learning approach
+
+#### Machine Learning approach
+
+Machine learning refers to giving computers ability to learn without explicitly being programmed.
+A machine learning approach uses machine learning algorithms to detect misinformation. Examples of these algorithms include:
+**Naïve Bayes:** uses probabilistic approaches based on Bayes theorem. This algorithm is often used for text classification.
+**Decision tree:** a supervised learning algorithm that has a tree-like flow. It helps in decision making. A useful algorithm for both classification and regression tasks.
+Random forest: simply a combination of decision trees.
+**Support Vector Machine:** a supervised learning algorithm. It examines data for classification and regression analysis. It classifies data into two categories.
+**Logistic regression:** contrary to the name, it is a classification algorithm used to estimate discrete values.
+**K-nearest-neighbour:** a simple algorithm that is used for both classification and regression tasks. Though it is more widely used for classification problems.
+
+Datasets are used to refine the algorithms. These datasets may be split as training data or test data.
+I have come across a lot of research where a system combines various machine learning algorithms and data mining. Often carried out on social media platforms, especially Twitter data. For example, a model may combine machine learning, through Naïve Bayes, Support Vector Machine, and Natural Language Processing to detect fake news. The accuracy of the results is usually determined by the combinations of models used and the datasets involved. A combination of available toolkits with Bayesian learning may be used to develop a fake news detector. These toolkits include Textblob, Natural Language and SciPy.
```

Comment:
**fix:**  and data mining. Often carried out

Reply:
For example, a model may combine machine learning, through Naïve Bayes, Support Vector Machine, and Natural Language Processing to detect fake news.


How would it go about doing this. Please expand on this sentence.